### PR TITLE
Publishing to maven central

### PR DESCRIPTION
**Pull request by christian-perez-adv**
This are the changes required to be able to publish the new OSS version of this SBT plugin to Maven Central.

Unfortunately we couldn't keep the previous organization ID, so users will have to update their projects to use the new one (`com.github.adevinta.unicron`). See the README for further information.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,65 +1,29 @@
...
-    dynverSonatypeSnapshots := true
-  )
-}
-
-lazy val bintraySettings = Seq(
-  publishMavenStyle := false,
-  bintrayOrganization := Some("adevinta-unicron"),
-  bintrayRepository := "sbt-plugins",
+  
+  name := "prometheus-datadog-bridge",
```
Comment (ekesken) [inline build.sbt]:
```suggestion
  name := "sbt-artifactory-settings",
```

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version = 1.3.2
\ No newline at end of file
+sbt.version = 1.4.7
```
Comment (ekesken) [inline project/build.properties]: ~ missing newline

File: src/main/scala/com/adevinta/unicron/artifactory/ArtifactorySettingsPlugin.scala [hunk 3]
```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2020 Adevinta
```
Comment (ekesken) [inline src/main/scala/com/adevinta/unicron/artifactory/ArtifactorySettingsPlugin.scala]: 2021?

**Review by ekesken — APPROVED** [review]

File: project/build.properties (hunk 2 above)
Comment (christian-perez-adv) [reply project/build.properties]: done

File: src/main/scala/com/adevinta/unicron/artifactory/ArtifactorySettingsPlugin.scala (hunk 3 above)
Comment (christian-perez-adv) [reply src/main/scala/com/adevinta/unicron/artifactory/ArtifactorySettingsPlugin.scala]: done

**Pull request merged**